move/axelar_gateway/sources/types/gateway_data.move

```diff
@@ -42,9 +44,30 @@ public struct GatewayDataV0 has store {
     operator: address,
     messages: Table<Bytes32, MessageStatus>,
     signers: AxelarSigners,
+    version_control: VersionControl,
 }
 
+// ------
+// Events
+// ------
+/// Emitted when a new message is sent from the SUI network.
+public struct ContractCall has copy, drop {
+    source_id: address,
+    destination_chain: String,
+    destination_address: String,
+    payload: vector<u8>,
+    payload_hash: address,
+}
 
+/// Emitted when a new message is approved by the gateway.
+public struct MessageApproved has copy, drop {
+    message: message::Message,
+}
+
+/// Emitted when a message is taken to be executed by a channel.
+public struct MessageExecuted has copy, drop {
+    message: message::Message,
+}
```

Comment:
we discussed moving the events to a dedicated module now. Move book mentions the following:
>The Sui Verifier requires the type passed to the emit function to be internal to the module. So emitting a type from another module will result in a compilation error. Primitive types, although they match the copy and drop requirement, are not allowed to be emitted as events.

It also means the event type name will change due to moving the module. This is a breaking change for amplifier/relayer, albeit a simple change to make.

Reply:
Let's use an `events` module so we can version it separately